### DISCUSSION TITLE
如何创建一个无需登录的项目或页面？

### DISCUSSION THREAD

**Discussion by zhaoyaohui0** [illacloud/illa-builder#1936] [Q&A]
如何做到发布后游客无需登录就可以查看页面？

**Comment by AruSeito** [comment]
您好，可以将该 APP 设置为 Public

**Reply by zhaoyaohui0** [reply]
不好意思，我没有找到可以设置APP为public的地方，也没有在使用说明中查找到相关的文档，能麻烦您详细说明下参数的位置吗？

**Reply by wenzhuo2022** [reply]
您好，您使用的是open source还是Cloud版本呢？这个功能目前只在Cloud版本里支持，如您需要使用的话，可以访问https://cloud.illacloud.com访问并使用Cloud版本

**Reply by macomzzl** [reply]
有计划在开源版本里添加这个功能吗@wenzhuo2022

**Reply by owenlongbo** [reply]
这会是一个Cloud版本功能，不会在开源版本里面提供权限控制相关的能力。
